cyctl/internal/get/modules.go

```diff
@@ -0,0 +1,37 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func ListModules(clientset *kubernetes.Clientset) {
+	labelSelector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"app.kubernetes.io/managed-by": "cyclops",
+		},
+	}
+
+	modules, err := clientset.AppsV1().Deployments("").List(context.TODO(), metav1.ListOptions{
```

Comment:
You are fetching for K8s deployments managed by Cyclops, but you will need to fetch Modules as CRDs.

I was looking into if you can use the [moduleset](https://github.com/cyclops-ui/cyclops/blob/f73ac60ec2ff796453536d5eace2a214a515e4a2/cyclops-ctrl/internal/cluster/v1alpha1/modules.go#L28) we already have and call for modules in a similar way you did with Deployments. I just need to refactor one part of cyclops-ctrl, and you can just import it and use it the same way it was used [here](https://github.com/cyclops-ui/cyclops/blob/aec912c35b5557eb2f72d5a08bf507e5dbbe5de2/cyclops-ctrl/internal/cluster/k8sclient/modules.go#L29)